tests/ttnn/integration_tests/yolox_m/README.md

```diff
@@ -0,0 +1,10 @@
+# Yolox_m ttnn implementation
+
+## Current status
+
+1. ttnn.conv in focus sub-module has PCC of 0.99. However, torch slice is used in this sub-module, because reference has this type of slice x[..., ::2, ::2] where ::2 indicates alternative elements will be chosen, and its throwing error in ::2.
```

Comment:
This README is out-dated. Please update with proper instructions on how to run the tests.
Also add a README for the demo separately. Refer to existing Demo README for any other model for reference.